CoastalResilienceNetwork/GeositeFramework
# Implement infographic support in the launchpad

## Overview

Add support for the infographic feature that was developed by TNC plugin authors.

Site admins can now enable this feature in the region config, provide a HTML template, and a button will be added to the launchpad plugin title. When users click this button, a modal infographic is displayed.

This implementation is largely based on the feature implemented in https://github.com/CoastalResilienceNetwork/ecological-effects-slc.

Connects to #939

### Demo

![tnc2](https://user-images.githubusercontent.com/1042475/32515781-2dfad5a8-c3cf-11e7-86b2-2356527b47d5.gif)

## Testing Instructions

- When the launchpad opens, click the infographic button, and verify the infographic modal opens.
- Disable the infographic in the config, and verify the app loads as normal, and the infographic button is not displayed.
